solidity/test/DepositRedemptionTest.js

```diff
@@ -234,15 +230,38 @@ describe("DepositRedemption", async function() {
       expect(tbtcOwed).to.eq.BN(depositValue)
     })
 
-    it("returns SignerFee if we are at-term, caller is TDT holder, and fee is not escrowed", async () => {
+    it("returns zero if we are at-term, caller is TDT holder, and fee is not escrowed", async () => {
       await increaseTime(depositTerm)
+      await feeRebateToken.burn(web3.utils.toBN(testDeposit.address), {
+        from: frtHolder,
+      })
+      const tbtcOwed = await testDeposit.getRedemptionTbtcRequirement.call(
+        tdtHolder,
+      )
+      expect(tbtcOwed).to.eq.BN(new BN(0))
```

Comment:
🤔 who pays the signers in this scenario? This disagrees with row 6 of [the spec’s table](https://docs.keep.network/tbtc/#_redemption_payment_and_disbursal_scenarios) and is because I did the flow diagram wrong. The TDT holder always owes the signer fee, even at term, if it’s not escrowed.

On the flip side, we can eliminate the FRT logic if we start adjusting the owed TBTC by the already-escrowed balance in the general case (as we’ve discussed doing for #680), since the existence of the FRT is only being used as a signal of whether or not the fee is escrowed. Relying on the actual escrow amount seems safer in the first place.